src/FSharp.Core/FSharp.Core.fsproj

```diff
@@ -4,7 +4,8 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(Configuration)' == 'Proto'">netstandard2.0</TargetFrameworks>
```

Comment:
Should we build ns21 for proto?

Reply:
@vzarytovskii No ... because the fsharp.build is loaded into msbuild, which is part of VS and can run on the desktop.